app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -93,6 +106,43 @@ class PatientSearchResultItemView(
     renderPatientPhoneNumber(searchQuery, model)
     renderVisited(model.lastSeen)
     renderLastSeen(model.lastSeen, currentFacilityId)
+    renderIdentifier(model.identifier, searchQuery)
+  }
+
+  private fun getId(identifier: Identifier, searchQuery: String): Id {
+    val id = when (identifier.type) {
+      BpPassport -> BpPassport.shortCode(identifier)
+      BangladeshNationalId -> identifier.value
+      EthiopiaMedicalRecordNumber -> identifier.value
+      IndiaNationalHealthId -> identifier.value
```

Comment:
Not necessary to change this, but these clauses could be collapsed into a single one.

```kotlin
BangladeshNationalId, EthiopiaMedicalRecordNumber, IndiaNationalHealthId -> identifier.value
```

Reply:
Omg, yes. Let me make this change.

Reply:
Changed it to 
```kt
BangladeshNationalId, EthiopiaMedicalRecordNumber, IndiaNationalHealthId, is Unknown -> identifier.value
```